src/utils.js

```diff
@@ -75,7 +75,7 @@ function getOptionsForFormatting(
 }
 
 function getRelevantESLintConfig(eslintConfig, eslintPath) {
-  const cliEngine = getESLintCLIEngine(eslintPath);
+  const cliEngine = getESLintCLIEngine(eslintPath, { ignore: false });
```

Comment:
Will this result in ignored files being formatted?

Reply:
No. It will just reduce the the unnecessary readings of the ignorefiles